### PR TITLE
Add arch property for windows_arm64 platform

### PR DESCRIPTION
**Pull request by keyonghan**
Window arm64 bots are using x64 built python, which gave incorrect arch info. This PR adds the arch info explicitly so that recipes can get this value and populate to benchmark tags.

Part of https://github.com/flutter/flutter/issues/135722

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -199,6 +199,8 @@ platform_properties:
       device_type: none
   windows_arm64:
     properties:
+      # The arch can be removed after https://github.com/flutter/flutter/issues/135722.
+      arch: arm64
```
Comment (loic-sharma) [inline .ci.yaml:203]: Should this be `arm` to match LUCI? See: https://github.com/luci/recipes-py/blob/0a42b2afe8ec90efadfd089f74459ae323c7ab30/recipe_modules/platform/api.py#L21

**Review by loic-sharma — APPROVED** [review]
Thanks for the work around, didn't know we could do this!

File: .ci.yaml (hunk 1 above)
Comment (keyonghan) [reply .ci.yaml]: Good catch! Updated.

**Review by cbracken — APPROVED** [review]
🎉

**Pull request merged**